spec/factories/transcriptions.rb

```diff
@@ -2,7 +2,347 @@
   factory :transcription do
     workflow
     group_id { "GROUP1A" }
-    text { { structure: "tbd" } }
+    text { {
```

Comment:
I think we probably want to avoid generating all of this text every time we create a transcription via factory. You can add a trait that will add all of this conditionally. Alternatively, put it in a fixture, then load it up and attach it for only the transcriptions that need to be fully honked up. Check out how I did project roles for an example of that way.

Reply:
Definitely - thanks for the tip, will do that.